.github/workflows/tests.yml

```diff
@@ -81,14 +81,14 @@ jobs:
       - name: "Compile OMP functions"
         run: inv -r faasmcli/faasmcli compile.user omp
       # --- Environment set-up ---
-      - name: "Set up test lib"
-        run: inv -r faasmcli/faasmcli libs.fake
       - name: "Run codegen"
         run: inv -r faasmcli/faasmcli codegen.local
       - name: "Run python codegen"
         run: inv -r faasmcli/faasmcli python.codegen
       - name: "Copy Python functions into place"
         run: inv -r faasmcli/faasmcli upload.user python --py --local-copy
+      - name: "Clear existing pyc files"
+        run: inv -r faasmcli/faasmcli python.clear-runtime-pyc
```

Comment:
This is a safety step that was missing before, in case pyc files are left over from the cpython container build